src/app/dashboard/Missions/MissionFundedStatusRow.jsx

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import { Mission } from "../../model";
+import AttachMoneyIcon from "@material-ui/icons/AttachMoney";
+import { Box, Grid } from "@material-ui/core";
+
+const Row = ({ children, Icon }) => {
+  if (!children) return null;
+  return (
+    <Grid container wrap="nowrap" alignItems="center">
+      <Box marginRight="5px" width="20px">
+        {Icon && <Icon color="primary" />}
+      </Box>
+      {children}
+    </Grid>
+  );
+};
+
+const MissionFundedStatusRow = ({ classes, mission }) => {
+  let missionFundedStatusText;
+  let fundedDateText = mission.fundedDate ? " on " + mission.fundedDate : "";
+  switch (mission?.fundedStatus) {
+    case Mission.FundedStatus.fundedbydonation:
+      missionFundedStatusText = "Funded By Donation ${fundedDateText}";
```

Comment:
Incorrect syntax here, these should be in backticks. `` and not quotes.

The linter is catching this warning too